extensions/amp-base-carousel/1.0/component.js

```diff
@@ -12,12 +12,9 @@ import {CarouselContext} from './carousel-context';
 import {ContainWrapper} from '#preact/component';
 import {Scroller} from './scroller';
 import {WithAmpContext} from '#preact/context';
-import {forwardRef, toChildArray} from '#preact/compat';
```

Comment:
Was there a reason we were explicitly importing toChildArray from preact/compat instead of Preact?
If not, LGTM

Reply:
from what i could tell, there didn't seem to be any good reason - the implementations are almost identical @alanorozco do you know why?

Reply:
No clue. Moving seems okay.